### PR TITLE
Branding in preparation for initial release

### LINKED ISSUE #3: Branding
Implement agreed branding (draft in progress from IFK Lund)

### PR DESCRIPTION
**Pull request by Linguisht**
Updating some formats in line with the graphical profile provided by IFK Lund and some images (purchased from Shutterstock -- is a eula required?). Changed to Arial based on the profile (does that need to be provided as a converted web font, or can one assume support?).

I'll add some content changes in a separate PR.

I guess this closes #3  (for now at least).
@peol -- for a live date *tomorrow* are #4 and #5 tricky? @qliklars has the domains for #4 but we don't have the target email for #5 yet.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
   padding: 0;
-  font-family: 'Open Sans', sans-serif;
+  font-family: 'Arial', sans-serif;
 }
 
 .hero {
   max-width:100%;
   width: 100vw;
   height: 100vh;
-  background-image: url('./alexander-redl-185764-unsplash.jpg');
```
Comment (peol) [inline src/style.css]: Remove this old file if it's no longer used

File: src/sparvagsstafetten_logotyp_white.svg [hunk 2]
```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (peol) [inline src/sparvagsstafetten_logotyp_white.svg]: Run SVG content through e.g. https://www.svgminify.com/ to reduce size

File: src/style.css [hunk 3]
```diff
@@ -1,14 +1,14 @@
 html, body, h1, h2, h3 {
   margin: 0;
   padding: 0;
-  font-family: 'Open Sans', sans-serif;
+  font-family: 'Arial', sans-serif;
```
Comment (peol) [inline src/style.css]: Arial should be considered safe to use without extra resources loaded, albeit ugly ;-)

File: src/style.css [hunk 4]
```diff
@@ -29,20 +29,45 @@ html, body, h1, h2, h3 {
...
 .button-register {
   padding: 1rem;
-  background-color: #77c277;
   margin: 1rem;
   display: inline-block;
   text-decoration: none;
-  color: #333;
+  border-radius: 1rem;
+  cursor: pointer;
+  color:#fff !important;
```
Comment (peol) [inline src/style.css]: Avoid `!important`, not sure why you'd need it here?

**Comment by peol** [comment]
@Linguisht 

> @peol -- for a live date tomorrow are #4 and #5 tricky? @qliklars has the domains for #4 but we don't have the target email for #5 yet.

#4 can be solved using a CNAME record, don't think you need anything else?
#5 is sorted elsewhere too, usually in the control panel where you registered the domain.

Unless I'm misunderstanding the question :-)

**Comment by Linguisht** [comment]
@peol: re domain questions, let's sync up with @qliklars tomorrow ; I'll address the other things right away

File: src/style.css (hunk 3 above)
Comment (Linguisht) [reply src/style.css]: Good, even though I agree

File: src/style.css (hunk 4 above)
Comment (Linguisht) [reply src/style.css]: Going for a short-cut to preserve the foreground on the button. I'll try a different angle. :)

File: src/style.css [hunk 5]
```diff
@@ -143,11 +148,7 @@ a:hover {
  cursor: pointer;
 }
 
-.navy {
-  background-color: #193970;
-  color: #e1e1e1;
-  box-shadow: inset 0 3px 3px #111;
-}
```
Comment (Linguisht) [inline src/style.css]: No longer used, so removed

**Comment by Linguisht** [comment]
@peol -- Anything else I should fix?

**Comment by peol** [comment]
LGTM

**Pull request merged**